src/operators.jl

```diff
@@ -858,3 +858,191 @@ function merge_vertices!(g::Graph{T}, vs::Vector{U} where U <: Integer) where T
 
     return new_vertex_ids
 end
+
+"""
+    linegraph(g)
+
+returns only the line graph of a given graph if returnmap is false
```

Comment:
We have the convention that we write these doc strings in the imperative tense, i.e you would write `Return line graph...?